tests/php/core-api/wpcom-fields/test_post-fields-publicize-connections.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+require_once JETPACK__PLUGIN_DIR . '/tests/php/lib/class-wp-test-jetpack-rest-testcase.php';
+require_once JETPACK__PLUGIN_DIR . '/tests/php/lib/class-wp-test-spy-rest-server.php';
+
+/**
+ * @group publicize
+ * @group rest-api
+ */
+class Test_WPCOM_REST_API_V2_Post_Publicize_Connections_Field extends WP_Test_Jetpack_REST_Testcase {
```

Comment:
Do you think it makes sense to have some tests for the cases when a user doesn't have the sufficient capabilities?

Reply:
Yes - good call. Noted as a TODO in https://github.com/Automattic/jetpack/pull/10605